config/api_versions.yml

```diff
@@ -0,0 +1,32 @@
+- v19.0
```

Comment:
Grabbed this list from Meta's [Graph API docs](https://developers.facebook.com/docs/graph-api/changelog/versions)